.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,62 @@
+name: Bug report
+description: Report an issue you encountered when playing OpenRCT2.
+labels: 
+- bug
+
+body:
+- type: markdown
+  attributes:
+    value: Before filing this bug, [please check if it hasn't already been reported](https://github.com/OpenRCT2/OpenRCT2/issues?q=is%3Aissue+is%3Aopen+label%3Abug). If it has, please add useful information to the existing issue instead.
+
+- type: input
+  attributes:
+    label: Operating System
+    placeholder: Windows 10, 64-bit
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: OpenRCT2 build
+    placeholder: OpenRCT2, v0.3.4.1 (d76611e on develop) provided by GitHub
+    description: You can copy the version information from the About window.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Describe the issue
+    description: Please describe the issue you are experiencing here.
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Area(s) with issue?
+    description: What things had an issue? Check all that apply.
```

Comment:
What or Which?

Reply:
Since it can be multiple things, what is correct.